### DISCUSSION TITLE
Express with Apollo Server Lambda

### DISCUSSION THREAD

**Discussion by deepinder10** [apollographql/apollo-server#5135] [Q&A]
Hello, I am trying to deploy my app on serverless but i made it with a middleware and apollo-server-express, now for deployment section to AWS lambda, the docs say to use apollo-server-lambda, but it doesn't expose the applyMiddleware method, how do i make use of that in my express app?

**Comment by glasser** [comment]
I'd personally suggest you try using https://www.npmjs.com/package/@vendia/serverless-express to convert your Express app to Lambda rather than use `apollo-server-lambda`, if you're actually making interesting use of Express. (This package is confusingly named and is unrelated to the `serverless` command line tool.)

If you'd like to use apollo-server-lambda, the entry point is called `createHandler` not `applyMiddleware` for Lambda. The way we document the different integrations of Apollo Server is not really ideal; tracking in #5097.

https://www.apollographql.com/docs/apollo-server/deployment/lambda/ shows how to deploy an apollo-server-lambda app with the `serverless` tool.
